compose/services/docker-compose.fuelrod-new.yml

```
@@ -33,16 +35,35 @@
     networks:
       - web
       - internal
+#    healthcheck:
```

yamllint (1.35.1)

[warning] comment not indented like content

(comments-indentation)